src/middleware/jsx/index.test.tsx

```diff
@@ -75,6 +75,50 @@ describe('render to string', () => {
     })
   })
 
+  describe('Special case for input "checked" and "selected" params', () => {
+    it('default prop value for checked', () => {
+      const template = <input type='checkbox' checked />
+      expect(template.toString()).toBe('<input type="checkbox" checked="checked"/>')
```

Comment:
I think it's better to render the attribute with no value like the below:

```
<input type="checkbox" checked/>'
```

Reply:
I think the following result is better here

```
<input type="checkbox" checked="" />
```

In #378, we made it possible to parse a rendered string as XML, just like in React, so I think it is better to adopt empty string for boolean attributes as well.

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes#boolean_attributes

Reply:
Alright! Let's go with `<input type="checkbox" checked=""/>`.
@digitalmio 
Is it OK?